nixos/modules/services/mail/postsrsd.nix

```diff
@@ -7,6 +7,19 @@
 let
 
   cfg = config.services.postsrsd;
+  runtimeDirectoryName = "postsrsd";
+  runtimeDirectory = "/run/${runtimeDirectoryName}";
+  # <<< TODO: follow RFC 42, need a libconfuse format first >>>
+  configFile = pkgs.writeText "postsrsd.conf" ''
+    secrets-file = "''${CREDENTIALS_DIRECTORY}/secrets-file"
+    domains = { "${cfg.domain}" }
```

Comment:
Can we change this to a list of str so that postfix' with multiple domains are supported? https://github.com/roehling/postsrsd/blob/main/doc/postsrsd.conf#L12

Reply:
Good idea, done! Given that other, backwards incompatible changes are occurring here, I've opted to remove the old option entirely.

Let me know how I should handle this? A release note? Something else?

Reply:
Yeah, we should do a release notes entry.

Reply:
I've added a release notes entry.